iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/methods/DeviceMethodTests.java

```diff
@@ -43,7 +44,7 @@ public void cleanToStart() throws Exception
     {
         super.cleanToStart();
     }
-
+/*
```

Comment:
what is this for?

Reply:
I forgot to undo that, but now it is gone